File: 8.py
# check if palindrome
num = int(input("Enter #:"))

temp = abs(num)
reverse = 0

while temp !=0 :
    reverse = (reverse * 10) + (temp % 10)
    temp = temp // 10

print(f"{num} Is Palindrome" if reverse == num else f"{num} Is Not Palindrome")

File: 8.c
// check if palindrome
#include <stdio.h>

int main(){
    int a, ld, rev=0, temp=0;
    printf("Enter number:");
    scanf("%d",&a);
    temp = a;
    
    while( temp != 0){
        ld = temp % 10;
        temp = temp / 10;
        rev  = rev * 10 + ld;
        
    }
    if (a == rev){
        printf("%d Is Palindrome",rev);
    }
    else{
        printf("%d Is Not Palindrome",rev);
    }
    return 0;
}